public/controllers/agent/components/agents-table.js

```diff
@@ -291,7 +288,7 @@ export const AgentsTable = withErrorBoundary(
         ip: agent.ip,
         status: agent.status,
         group_config_status: agent.group_config_status,
-        group: checkField(agent.group),
+        group: agent?.group || [],
```

Comment:
**thought**: is this change breaking the render of the group? 

https://github.com/wazuh/wazuh-kibana-app/blob/fe811908c683a25c85ad0acf891766f6c26bf2b2/public/controllers/agent/components/agents-table.js#L481

We should review the cases and decide what displays when there is not a group. If this change doesn't break the funcitonality, then we could remove the check of if the group is `-`.

Reply:
Good catch! Fixed it in https://github.com/wazuh/wazuh-kibana-app/pull/4828/commits/6e47fe8914d1f79f30d90876065ba8dd58714f17

![Screenshot from 2023-01-19 12-20-53](https://user-images.githubusercontent.com/9343732/213430839-371333aa-83e4-4603-950e-140219a07aa5.png)